docs/developing/get-started.md

```diff
@@ -139,13 +96,15 @@ etherscan: {
 The command for verifying a deployed contract is:
 
 ```jsx
-./node_modules/.bin/hardhat verify <DEPLOYED_CONTRACT_ADDRESS> --network neonevm
+npx hardhat verify <DEPLOYED_CONTRACT_ADDRESS> --network neondevnet
 ```
 
-Follow this GitHub tutorial for contract verification [github.com/neonscan/hardhat-integrate](https://github.com/neonscan/hardhat-integrate).
+Follow this GitHub tutorial for contract verification [Hardhat Github Tutorial](https://github.com/neonlabsorg/neon-tutorials/tree/main/hardhat).
 
 ## Gotchas
 
+### Reaching account limits
+
 You may encounter an error `The transaction requires too lot of accounts`, i.e. that the transaction has exceeded the account limit. This error refers to the fact that the transaction on Solana does not allow the number of accounts to be more than 64. If you do need more than 64 accounts, the smart contract function throwing the error needs to be optimized.
```

Comment:
@sukanyaparashar Please can you trigger this error and see if the team have improved the English in the error message yet, I did ask for an update >> IF so, can you grab the text and replace?

Reply:
Let me see if I can reproduce this error in some example.